### PR TITLE
Added break statement.

### LINKED ISSUE #6209: update isValid() in KeyUtils
### Problem

check ThresholdKey needs to break after finding one invalid key.

### Solution

fix in the code

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by iwsimon**
**Description**:

**Related issue(s)**:

Fixes #6209 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by povolev15 — APPROVED** [review]
LGTM

File: hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/key/KeyUtils.java [hunk 1]
```diff
@@ -99,6 +99,7 @@ public static boolean isValid(@Nullable final Key pbjKey) {
             for (Key keys : ((ThresholdKey) key.value()).keys().keys()) {
                 if (!isValid(keys)) {
                     isKeyListValid = false;
+                    break;
```
Comment (netopyr) [inline hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/key/KeyUtils.java:102]: Guess you could also just return `false` here.

**Review by netopyr — APPROVED** [review]

File: hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/key/KeyUtils.java (hunk 1 above)
Comment (iwsimon) [reply hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/key/KeyUtils.java:102]: No, we have this check after break: return (threshold >= 1 && threshold <= length && isKeyListValid);

Comment (netopyr) [reply hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/key/KeyUtils.java:102]: Yes and `isKeyListValid` will be `false`. Therefore we will return `false` anyway.

But it does not really matter. It is just a minor change that has no real effect. Just go ahead and change it in any way you like.

**Review by Neeharika-Sompalli — APPROVED** [review]

**Pull request merged**